lib/task-data/tasks/dell-racadm-get-firmware-list-catalog.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'dell racadm get firmware list catalog',
+    injectableName: 'Task.Dell.Racadm.GetFirmwareListCatalog',
+    implementsTask: 'Task.Base.Dell.Racadm.Catalog',
+    optionsSchema: 'dell-racadm-catalog.json',
+    options: {
+        action: 'getSoftwareList'
```

Comment:
@anhou at a high level the intention of this command is understood. I will make sure the naming is apt in future, for now I feel there's a risk to go back and rename terms as the PRs are close to getting merged.